### PR TITLE
mobile wallet options

### PR DESCRIPTION
**Pull request by rolandpo**
issue #865

- grey out Ledger option on mobile wallet modal
- on clicking Ledger prompt user to use WalletConnect with Ledger Live instead
- add MetaMask deeplink that takes user to app or appstore depending on whether it's installed

### CODE REVIEW

File: dapp/src/components/WalletSelectContent.js [hunk 1]
```diff
@@ -107,8 +114,19 @@ const WalletSelectContent = ({}) => {
...
               key={name}
-              className="connector-button d-flex align-items-center"
-              onClick={() => onConnect(name)}
+              className={`connector-button d-flex align-items-center ${
+                isMobile && name === 'Ledger' ? 'grey' : ''
+              }`}
+              onClick={() => {
+                if (isMobile && name === 'MetaMask') {
+                  setError(null)
+                  window.location.href = process.env.METAMASK_DEEPLINK
```
Comment (sparrowDom) [inline dapp/src/components/WalletSelectContent.js:123]: staging / prod / local environment should have this env var available

**Review by sparrowDom — COMMENTED** [review]
Left 1 comment please take a look. Also curious have you found a convenient way to test this? Maybe deploy to staging and test with a mobile device?

File: dapp/src/components/WalletSelectContent.js (hunk 1 above)
Comment (rolandpo) [reply dapp/src/components/WalletSelectContent.js:123]: yep, forgot to add

**Comment by rolandpo** [comment]
added MetaMask deeplink to dapp .env file. I was testing the mobile stuff with localtunnel to generate a link for mobile access

**Comment by sparrowDom** [comment]
Tested this a couple of comments: 
 - when on mobile device can you change the order of connections to: WalletConnect, CoinbaseWallet, MyEtherWallet, Metamask, grayed out Leger. It makes more sense to me to put the popular/convenient option on top. And those differ between desktop / mobile devices
 - when clicking on metamsk link on mobile device this error page opens for me
Screenshot_20220617-131855

**Comment by rolandpo** [comment]
not sure about this error page, it works for me on iphone on safari/chrome

**Comment by sparrowDom** [comment]
This is usual behaviour where Android / iPhone devices behave differently. Do you have an Android device you can use to test this? If not there should be Android simulators available.

**Comment by rolandpo** [comment]
changed the order of wallet options for mobile

**Comment by sparrowDom** [comment]
Issue still persists for me on Android, but it is not a blocker to get this merged in. Great work thanks 🙏

**Review by sparrowDom — APPROVED** [review]
LGTM

**Pull request merged**